### PR TITLE
Add rake tasks

### PR DESCRIPTION
**Pull request by y9v**
This PR adds two rake tasks as described in https://cultofmartians.com/tasks/logux-rails-rake.html.

```sh
⋊> bundle exec rake logux:actions
   action.type Class#method
blog/notes/add Actions::Blog::Notes#add
   comment/add Actions::Comment#add
```

```sh
⋊> bundle exec rake logux:channels
channel Class
   post Channels::Post
```

### CODE REVIEW

**Comment by lebedev-yury** [comment]
> There is still a failing test example, when running the full suite (the output of `rake logux:actions` contains a class which was dynamically created in another test example).
> 
> I will fix this issue tomorrow, it seems that the object is not removed from `ObjectSpace` after the constant removal (manually triggering GC doesn't help either)

Fixed it, it's removed from ObjectSpace if we remove it's methods first.

**Comment by wilddima** [comment]
Thank you for the pull request, I'm gonna check it soon.

**Comment by ymn** [comment]
@wilddima I still need to add support for namespaced actions and channels. I'll ping you when it's ready

**Comment by ymn** [comment]
@wilddima the issues are fixed; I also made the output a bit prettier

File: README.md [hunk 1]
```diff
@@ -38,6 +38,8 @@ After this, POST requests to `/logux` will be processed by LoguxController. You
 
 LoguxRails will try to find Action for the specific message from logux-server. For example, for `project/rename` action, you should define `Action::Project` class, inherited from `Logux::Action` base class, and implement `rename` method.
 
+You can execute `rake logux:actions` to get the list of available action types, or `rakelogux:channels` to get the list of available channels
```
Comment (wilddima) [inline README.md]: Typo `rakelogux:channels` 🙂

Comment (ymn) [reply README.md]: thanks, fixed

**Comment by ymn** [comment]
@wilddima thank you!

**Pull request merged**